flytekit/core/reference_entity.py

```diff
@@ -114,7 +113,7 @@ def unwrap_literal_map_and_execute(
 
         # Invoked before the task is executed
         # Translate the input literals to Python native
-        native_inputs = TypeEngine.literal_map_to_kwargs(ctx, input_literal_map, self.interface.inputs)
+        native_inputs = TypeEngine.literal_map_to_kwargs(ctx, input_literal_map, self._native_interface.inputs)
```

Comment:
Another note for @wild-endeavor, this method looks repeated from the task/workflow - call method? Do you think its better to factor it out?

Reply:
Yeah but maybe not as part of this pr.  All of reference entity is copy/paste from other components - we need to clean up things generally.